sub-packages/bionemo-esm2/tests/bionemo/esm2/model/finetune/test_finetune_regressor.py

```diff
@@ -27,7 +27,9 @@
 
 
 # To download a 8M internally pre-trained ESM2 model
-pretrain_ckpt_path = load("esm2/nv_8m:2.0")
+@pytest.fixture
+def pretrain_ckpt_path():
```

Comment:
maybe it can be moved to conftest?

Reply:
I actually went the other way and just moved the load calls into the tests themselves. I don't really like fixtures; they sort of obfuscate where the data is coming from, and in this case the `load()` call itself is less code than having to pass that fixture around